### ISSUE TITLE
Upstream Mastodon 4.2 has critical security vulnerability

### ISSUE THREAD

**Issue by dankar** [bitnami/charts#23030]
### Name and Version

bitnami/mastodon 4.2.3

### What architecture are you using?

None

### What steps will reproduce the bug?

Yesterday mastodon released 4.2.5 which fixes a critical security vulnerability. (CVE-2024-23832)

This might be the wrong way to report this, but I figured I'd let the maintainers here know about this in case a new chart release is not already on the way.

### What do you see instead?

I see mastodon 4.2.4 instead of 4.2.5 :)

**Comment by kznrluk** [comment]
Essentially, if the bitnami/mastodon image could be updated, specifying the image name through values.yaml should resolve the issue. However, as of now, the latest image is 4.2.4, so that is not possible. We are waiting for the image to be updated.

**Comment by carrodher** [comment]
Thank you for bringing this to our attention. Our automated test and release pipeline detected the new version some hours ago and it is currently being built and tested. If everything works fine, a new container image and Helm chart bundling the latest version should be available soon.

**Comment by ajguyot** [comment]
For anyone waiting on this, fyi the image was updated ~8 hours ago and is available. Not sure what the automated delay is before a new chart version will drop, but I just patched my instance using the current version of this chart (4.2.3) and the new version of the image: `4.2.5-debian-11-r0`.

Image PR here: https://github.com/bitnami/containers/pull/55998

No major version bumps on dependencies so I don't expect any issues with upgrading before the chart update drops.
